spec/index.bs

```diff
@@ -980,6 +980,34 @@ The default is used if a value is not explicitly specified in {{GPUDeviceDescrip
         <tr><th>Limit name <th>Type <th>[=limit/Better=] <th>[=limit/Default=]
     </thead>
 
+    <tr><td><dfn>maxTextureDimension1D</dfn>
+        <td>{{GPUSize32}} <td>Higher <td>8192
+    <tr class=row-continuation><td colspan=4>
+        The maximum allowed {{GPUTextureDescriptor/size}}.{{GPUExtent3DDict/width}}
+        for which {{GPUTextureDescriptor/dimension}} is {{GPUTextureDimension/"1d"}}
+        when creating a {{GPUTexture}}.
+
+    <tr><td><dfn>maxTextureDimension2D</dfn>
+        <td>{{GPUSize32}} <td>Higher <td>8192
+    <tr class=row-continuation><td colspan=4>
+        The maximum allowed {{GPUTextureDescriptor/size}}.{{GPUExtent3DDict/width}} and {{GPUTextureDescriptor/size}}.{{GPUExtent3DDict/height}}
+        for which {{GPUTextureDescriptor/dimension}} is {{GPUTextureDimension/"2d"}}
+        when creating a {{GPUTexture}}.
+
+    <tr><td><dfn>maxTextureDimension3D</dfn>
+        <td>{{GPUSize32}} <td>Higher <td>1024
```

Comment:
Let's raise this to 2048. Almost no devices have fewer; the only reports I see are for some old mobile and some Intel chips on Linux that most likely have old or weird drivers (as there are other reports on the same hardware that support 2048).

Reply:
Agreed. Done!